_generator/types.yaml

```
@@ -1780,3 +1780,93 @@ types:
   departmentfilterparameters:
     file: departments.md
     anchor: departmentfilterparameters
+  billingautomationresult:
+    file: billingautomations.md
+    anchor: billing-automation-result
+  billingautomation:
+    file: billingautomations.md
+    anchor: billing-automation
+  companywithbillingautomationrelation:
+    file: billingautomations.md
+    anchor: company-with-billing-automation-relation
+  companyrelationsparameters:
+    file: billingautomations.md
+    anchor: company-with-relations-parameters
+  billingautomationprepaymentenum:
+    file: billingautomations.md
+    anchor: billing-automation-prepayment-type
+  billingautomationassignmenttargettypeenum:
+    file: billingautomations.md
+    anchor: billing-automation-assignment-target-type
+  billingautomationtriggertypeenum:
+    file: billingautomations.md
+    anchor: billing-automation-trigger-type
+  billingautomationbillaggregationtypeenum:
+    file: billingautomations.md
+    anchor: billing-automation-bill-aggregation-type
+  billingautomationorderitemconsumptionperiodtypeenum:
+    file: billingautomations.md
+    anchor: billing-automation-order-item-consumption-period-type
+  billingautomationassignment:
+    file: billingautomations.md
+    anchor: billing-automation-assignment
+  billingautomationitemtypesparameters:
+    file: billingautomations.md
+    anchor: billing-automation-item-types-parameters
+  billingautomationaccountingcategoryassignment:
+    file: billingautomations.md
+    anchor: billing-automation-accounting-category-assignment
+  accountingcategoryrouteditemtypes:
+    file: billingautomations.md
+    anchor: accounting-category-routed-item-types
+  billingautomationfilterparameters:
+    file: billingautomations.md
+    anchor: billing-automation-filter-parameters
+  billingautomationmodificationresult:
+    file: billingautomations.md
+    anchor: billing-automation-modification-result
+  multiplebillingautomationaddparameters:
+    file: billingautomations.md
+    anchor: multiple-billing-automation-add-parameters
+  billingautomationaddparameters:
+    file: billingautomations.md
+    anchor: billing-automation-add-parameters
+  billingautomationassignmentaddparameters:
+    file: billingautomations.md
+    anchor: billing-automation-assignment-add-parameters
+  multiplebillingautomationupdateparameters:
+    file: billingautomations.md
+    anchor: multiple-billing-automation-update-parameters
+  billingautomationupdateparameters:
+    file: billingautomations.md
+    anchor: billing-automation-update-parameters
+  companywithbillingautomationrelationienumerableupdatevalue:
+    file: billingautomations.md
+    anchor: companywithbillingautomationrelationienumerableupdatevalue
```

Update value type names need to be overridden in this file because it is not possible to specify the name in the source code annotations.

```suggestion
  companywithbillingautomationrelationienumerableupdatevalue:
    file: billingautomations.md
    anchor: companywithbillingautomationrelationienumerableupdatevalue
    titleOverride: Company with billing automation relations update value
```